docs/tutorial/part-zero/index.md

```diff
@@ -211,7 +211,7 @@ Now you are ready to use the Gatsby CLI tool to create your first Gatsby site. U
 2. Create a new site from a starter:
 
 ```shell
-gatsby new hello-world https://github.com/gatsbyjs/gatsby/tree/master/starters/hello-world
+gatsby new my-hello-world-starter https://github.com/gatsbyjs/gatsby-starter-hello-world
```

Comment:
please leave the folder name as is, otherwise the folder name `hello-world` in line 220 and 228 needs also a change

```suggestion
gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world
```